content/en/blog/2023/announcing-notation-rc5.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Notation v1.0.0-RC.5 is available!
+author:  "Notation Release Team"
+date: 2023-05-18
+draft: false
+---
+
+The Notation maintainers are pleased to announce v1.0.0-RC.5 for [notation CLI](https://github.com/notaryproject/notation) and [notation-go library](https://github.com/notaryproject/notation-go). This blog walks you through the major updates in this release.
+
+## Major updates in notation and notation-go
+
+Notation is a CLI tool to sign and verify OCI artifacts. The v1.0.0-RC.5 changes include but are not limited to the following. See the [v1.0.0-RC.5 release notes](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.5) for details.
+
+* Improve the output message when there is no signature associated with the image being referred
+* Add compatibility  with the [OCI distribution-spec 1.1.0-RC.2](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0-rc2) and [OCI image-spec v1.1.0-RC.3](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc.3)
```

Comment:
Change "add Combability" to "Update compatibility"  I say this because, we had prior compatibility that we just updated to meet the latest revisision